test/libsolidity/SemanticTest.cpp

```diff
@@ -579,17 +584,23 @@ bool SemanticTest::checkGasCostExpectation(TestFunctionCall& io_test, bool _comp
 		!m_enforceGasCost ||
 		m_gasUsed < m_enforceGasCostMinValue ||
```

Comment:
Note that gas enforcement threshold is still based on the total and does not exclude code deposit cost. I'm not really sure which is better, but neither seems wrong either so I kept it at that.